test/functional/services/remote/leadfoot_command.js

```diff
@@ -8,15 +8,24 @@ const SECOND = 1000;
 const MINUTE = 60 * SECOND;
 
 let attemptCounter = 0;
-async function attemptToCreateCommand(log, server, chromedriverApi) {
+async function attemptToCreateCommand(log, server, driverApi) {
   const attemptId = ++attemptCounter;
+  let capabilities = {};
+
+  switch(driverApi.getBrowserName()) {
```

Comment:
How about moving this into `driverApi.getRequiredCapabilites()`, rather than exposing the name and expecting this module to know about the full list of possibilities?

Reply:
Done.